website/settings/celeryconfig.py

```diff
@@ -0,0 +1,34 @@
+from celery.schedules import crontab
+
+
+##### Celery #####
+## Default RabbitMQ broker
+BROKER_URL = 'amqp://'
+
+# Default RabbitMQ backend
+CELERY_RESULT_BACKEND = 'amqp://'
+
+# CELERYBEAT_SCHEDULE_FILENAME = 'celeryconfig.py'
+# Setting up a scheduler
+CELERYBEAT_SCHEDULE = {
+    '5-minute-emails': {
+        'task': 'notify.send_users_email',
+        'schedule': crontab(minute='*/5'),
```

Comment:
Sorry, can update the document. There were discussions that 1 minute was too quick, 15 too long.

Also, are we supposed to post proposals?